scene/2d/parallax_layer.cpp

```diff
@@ -34,6 +34,9 @@
 
 void ParallaxLayer::set_motion_scale(const Size2 &p_scale) {
 
+	if (!get_parent())
+		return;
+
 	motion_scale = p_scale;
```

Comment:
As mentioned on IRC, I actually wonder now if we don't want to allow setting the property before returning. I'm not sure exactly how ParallaxLayer and ParallaxBackground have to interact with each other here, it would need to be double checked.